.golangci.yml

```diff
@@ -57,78 +77,50 @@ linters-settings:
     simple: true
     range-loops: true
     for-loops: false
+  gosec:
+    excludes:
+      - G115
```

Comment:
Can we disable it only for specific line/file?

Reply:
Done